### PR TITLE
Add the initial support for cgroupv2

### PR DESCRIPTION
**Pull request by avagin**

### CODE REVIEW

File: test/zdtm.py [hunk 1]
```diff
@@ -1645,7 +1676,11 @@ def get_freezer(desc):
         return noop_freezer()
 
     fd = desc.split(':')
-    fr = cg_freezer(path=fd[0], state=fd[1])
+
+    if os.access("/sys/fs/cgroup/user.slice/cgroup.procs", os .F_OK):
```
Comment (Snorch) [inline test/zdtm.py:1680]: Are unified cgroups always mounted just in /sys/fs/cgroup, not /sys/fs/cgroup/unified? (I mean one can do something like https://blog.christophersmart.com/2019/12/15/enabling-docker-in-fedora-31-by-reverting-to-cgroups-v1/)

**Comment by adrianreber** [comment]
This PR fixes for me the problems reported in https://github.com/checkpoint-restore/criu/issues/1034

Using Podman and crun with CRIU and this PR I can checkpoint and restore a container multiple times using the cgroup2 freezer.

Thanks.

File: test/zdtm.py (hunk 1 above)
Comment (adrianreber) [reply test/zdtm.py:1680]: Starting with Fedora 31 there is only cgroup2 available and it is mounted at /sys/fs/cgroup. I expect that every systemd based distribution will mount cgroup2 at that location if it switches to cgroup2 only.

**Comment by avagin** [comment]
@adrianreber With this pr, criu creates cgroups and moves tasks in them, but it doesn't restore any cgroup properties (memory.max, cpu.weight, etc). If you want, you can start working on restoring these properties. I don't have a plan to work on this in the near future.

**Comment by adrianreber** [comment]
Do you want to merge this PR or wait for complete cgroup2 support based on this PR? Having this PR merged would be an improvement to the current code.

File: test/zdtm.py (hunk 1 above)
Comment (avagin) [reply test/zdtm.py:1680]: @Snorch In your example, cgroupv2 will be mounted in /sys/fs/cgroup/unified, but the freezer controller will be mounted separately in /sys/fs/cgroup/freezer/. I have checked and my changes works fine in this case.

**Pull request merged**